libcontainer/nsenter/nsexec.c

```diff
@@ -132,9 +123,6 @@ int setns(int fd, int nstype)
 }
 #endif
 
-/* XXX: This is ugly. */
-static int syncfd = -1;
```

Comment:
This is an example that could be a different commit, IMHO. It can be of the last ones, if that helps to avoid breaking compilation/tests. But if you split this out in another commit and just mention that you move to function X now that YY happens and we don't need it here anymore.

If I want to review in detail, it's hard when logically/semantically unrelated changes are mixed together. Because I go and check if this is sound, but checking that can be several steps, and then I have to do backtracking and come back to this point and continue reviewing. It's way more likely I'll miss more stuff if I review this way.

On the other hand, if this was a different commit and you call out in the commit msg why is this possible now, then it will be very easy to review.